game-core/src/main/java/games/strategy/triplea/delegate/battle/AirBattle.java

```diff
@@ -910,11 +911,9 @@ private void remove(
       return;
     }
     final Collection<Unit> killed = getUnitsWithDependents(killedUnits);
-    final Change killedChange = ChangeFactory.removeUnits(battleSite, killed);
-    final String transcriptText =
-        MyFormatter.unitsToText(killed) + " lost in " + battleSite.getName();
-    bridge.getHistoryWriter().addChildToEvent(transcriptText, new ArrayList<>(killed));
-    bridge.addChange(killedChange);
+    final KillUnits killUnits = new KillUnits(battleSite, killed);
```

Comment:
nit, unnecessary intermediate variable. Can be a single line
```
new KillUnits(battleSite, killed).perform(bridge);
```

Side-note: I think the above does reveal a code smell where the bridge should be in control of when the change happens. EG:
```
bridge.addChange(new KillUnits(battleSite, killed);
```

Reply:
The plan is to return these history change objects to the caller so that they can then pass it to the bridge.  So, there is still a bit of code smell because this is a partial step.